db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -309,10 +349,49 @@ class NonBatchedOpsStressTest : public StressTest {
         std::terminate();
       }
     }
+    if (fault_fs_guard) {
+      fault_fs_guard->DisableErrorInjection();
+    }
 #endif // NDEBUG
 
-    for (const auto& s : statuses) {
-      if (s.ok()) {
+    for (size_t i = 0; i < statuses.size(); ++i) {
+      Status s = statuses[i];
+      bool is_consistent = true;
+      // Only do the consistency check if no error was injected and MultiGet
+      // didn't return an unexpected error
+      if (do_consistency_check && !error_count &&
+            (s.ok() || s.IsNotFound())) {
+        Status tmp_s;
+        std::string value;
+
+        tmp_s = db_->Get(readoptionscopy, cfh, keys[i], &value);
+        if (!s.ok() && tmp_s.ok()) {
+          fprintf(stderr, "MultiGet returned different results with key %s\n",
+                  keys[i].ToString(true).c_str());
+          fprintf(stderr, "Get returned ok, MultiGet returned not found\n");
+          is_consistent = false;
+        } else if (s.ok() && tmp_s.IsNotFound()) {
+          fprintf(stderr, "MultiGet returned different results with key %s\n",
+                  keys[i].ToString(true).c_str());
+          fprintf(stderr, "MultiGet returned ok, Get returned not found\n");
+          is_consistent = false;
+        } else if (s.ok() && value != values[i].ToString()) {
+          fprintf(stderr, "MultiGet returned different results with key %s\n",
```

Comment:
I think we need to check for error status from Get, even though we're outside error injection region. We at least want to see what that status is if it's an error, and hypothetically, the "correct" value could be empty string (which would currently mask an error from Get).